.github/workflows/codspeed.yml

```
@@ -46,9 +46,9 @@ jobs:
       - name: Build Yak
         run: pnpm run build
       - name: Build benchmark suite
-        run: cd packages/benchmark && pnpm run codspeed 
+        run: cd packages/benchmark && pnpm run codspeed
       - name: Run benchmarks
-        uses: CodSpeedHQ/action@v1
+        uses: CodSpeedHQ/action@v2
```

I do not see the debug logs in the workflow 😖 
Can you add the env variable like so?

```suggestion
        uses: CodSpeedHQ/action@v2
        env:
          CODSPEED_DEBUG: true
```

---

I've added it. Thanks for looking into it!